saladItem.js

```diff
@@ -1,3 +1,68 @@
 import React, { useReducer, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { createUseStyles } from 'react-jss';
+
+import UserContext from '../User/User';
+import { SaladContext } from '../SaladMaker/SaladMaker';
+
+// Items for a list of Salad options for the Salad Maker Interface
+const useStyles = createUseStyles({
+        add: {
+          background: 'none',
+          border: 'none',
+          cursor: 'pointer',
+        },
+            favorite: {
+            fontSize: 20,
+            position: 'absolute',
+            top: 10,
+            right: 10,
+            },
+        image: {
+          fontSize: 80
+        },
+    wrapper: {
+        border: 'lightgrey solid 1px',
+        margin: 20,
+        padding: 25,
+        position: 'relative',
+        textAlign: 'center',
+        textTransform: 'capitalize',
+        width: 200,
+    }
+});
+
+var reducer = 0;
+const reducer = key => key + 1;
+// Provides a viewable list of Salad Items by Image and Name
+export default function SaladItem({ image, name }) {
+  const classes = useStyles();
+  const { setSalad } = useContext(SaladContext)
+  const user = useContext(UserContext);
+  const favorite = user.favorites.includes(name);
+  const [id, updateId] = useReducer(reducer, 0);
+  function update() { 
+      setSalad({ name, id: `${name}-${id}`})
+        updateId();
+    };
+
+  return(
+    <div className={classes.wrapper}>
+        <h3>
+          {name}
+        </h3>
+        <span className={classes.favorite} aria-label={favorite ? 'Favorite' : 'Not Favorite'}>
+          {favorite ? '😋' : ''}
+        </span>
+        <button className={classes.add} onClick={update} onTouchMove={}>
+          <span className={classes.image} role="img" aria-label={name}>{image}</span>
+        </button>
+        <h3><!-- spacer --></h3>
```

Comment:
I'm really not sure what the purpose of this tag is, but it is causing an error so should be removed. If trying to add space below the button, you can add a margin/padding to useStyles for add or wrapper.